<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BloodSpeedAdjust</class>
 <widget class="QWidget" name="BloodSpeedAdjust">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widgetAdd" native="true">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>0</y>
     <width>110</width>
     <height>125</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>125</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>110</width>
     <height>125</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <widget class="QPushButton" name="pushButtonAdd">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>110</width>
      <height>125</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>125</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>125</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButtonAdd{
	background-image: url(:/public/right/speed_add.png);
	border:none;
}
#pushButtonAdd:pressed{
	background-image: url(:/public/right/speed_add_clicked.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdd">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>34</y>
      <width>56</width>
      <height>56</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/public/right/speed_adding.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetSub" native="true">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>305</y>
     <width>110</width>
     <height>125</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>125</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>110</width>
     <height>125</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <widget class="QPushButton" name="pushButtonSub">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>110</width>
      <height>125</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>125</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>125</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButtonSub{
	background-image: url(:/public/right/speed_minus.png);
	border:none;
}
#pushButtonSub:pressed{
	background-image: url(:/public/right/speed_minus_clicked.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSub">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>55</y>
      <width>51</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>14</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>14</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/public/right/speed_minusing.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetValue" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>240</width>
     <height>210</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>210</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>210</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#widgetValue{
	background-image: url(:/public/right/bloodmotor.png);
}</string>
   </property>
   <widget class="QLabel" name="labelValue">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>43</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:48px;
color:#fff;</string>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelUnit">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>99</y>
      <width>150</width>
      <height>44</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>44</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>44</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20px;
color:#ffffff;</string>
    </property>
    <property name="text">
     <string>ml/min
血流量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>144</y>
      <width>150</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>22</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#fff;
font-size:20px;</string>
    </property>
    <property name="text">
     <string>φ8mm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
